grouper/api/handlers.py

```diff
@@ -93,6 +94,20 @@ def on_finish(self):
         stats.log_rate("response_status_{}".format(response_status), 1)
         stats.log_rate("response_status_{}_{}".format(self.__class__.__name__, response_status), 1)
 
+    def log_exception(
+        self,
+        exc_type,  # type: Optional[Type[BaseException]]
+        exc_value,  # type: Optional[BaseException]
+        exc_tb,  # type: Optional[TracebackType]
+    ):
+        # type: (...) -> None
+        if isinstance(exc_value, HTTPError):
```

Comment:
Using `exc_type` confuses mypy because it doesn't know to infer that `exc_value` therefore has a `status_code` element. Yes, there's no purpose in including `exc_type` other than that Python always does, but that seemed like a good enough reason since this is an external API.